teaching_platform/main.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import yaml
+import jsonpickle
+from flask import Flask
+
+from teaching_platform.users.controller import user_controller_bp
+from teaching_platform.lessons.controller import lesson_controller_bp
+from teaching_platform.projects.controller import project_controller_bp
+from teaching_platform.extensions import db
+
+
+with open("../config.yaml", "r") as f:
+    config = yaml.load(f, Loader=yaml.FullLoader)
+db_config = config["database"]
+
+logging.basicConfig(level=logging.INFO, filemode="w", filename="../logs.log")
```

Comment:
I advise to make logging level configurable.

Reply:
Done

Reply:
Done